app/assets/javascripts/bastion/components/notification.service.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.components.service:Notification
+ *
+ * @description
+ *  Service to display a foreman toast notification
+ */
+angular.module('Bastion.components').service("Notification", ['$interpolate', 'foremanReact', function ($interpolate, foremanReact) {
+    function interpolateIfNeeded(message, context) {
+        var result = message;
+
+        if (context) {
+            result = $interpolate(message)(context);
+        }
+
+        return result;
+    }
+
+    this.setSuccessMessage = function (message, context) {
+        foremanReact.toastNotifications.notify({message: interpolateIfNeeded(message, context), type: 'success'});
```

Comment:
nitpick: but this doesnt actually go to the "react/redux" interface vs to the "legacy" tfm.toastNotifications name same, hence maybe it should not be called react. (to confuse other future readers).

Reply:
That's a good point, will update.